keras_nlp/models/gpt_neo_x/gpt_neo_x_attention.py

```diff
@@ -14,7 +14,7 @@
 import tensorflow as tf
 from tensorflow import keras
 
-from keras_nlp.models.gpt_neo_x.rotary_embedding import RotaryEmbedding
```

Comment:
one more nit, let's pass the arguments to RotaryEmbedding via keyword args, not positionally, below.